diff --git a/app/presenters/user_presenter.rb b/app/presenters/user_presenter.rb
index b09f1f79a..1b57078d5 100644
--- a/app/presenters/user_presenter.rb
+++ b/app/presenters/user_presenter.rb
@@ -58,7 +58,7 @@ class UserPresenter
   end
 
   def favorites
-    ids = Favorite.select(:post_id).where(user_id: user.id).order(created_at: :desc).limit(50).map(&:post_id)[0..5]
+    ids = Favorite.select(:post_id).where(user_id: user.id).order(id: :desc).limit(50).map(&:post_id)[0..5]
     Post.where(id: ids)
   end
 
